proto/brpb.proto

```diff
@@ -246,6 +247,11 @@ message CloudDynamic {
     map<string, string> attrs = 3;
 }
 
+// HDFS storage backend saves file into HDFS compatible storages
+message Hdfs {
+    string remote = 1;
```

Comment:
```suggestion
    string path = 1;
```

Reply:
This parameter is currently designed to be a URL: `hdfs:///some/path` or `hdfs://host:port/some/path`

Is `path` a suitable name?

Reply:
add a comment to explain the details?